internal/middlewares.go

```diff
@@ -32,7 +31,11 @@ func AuthMiddleware(next http.Handler) http.Handler {
 
 		bearerToken, ok := r.Header[authHeader]
 		if !ok {
-			sendUnauthorizedResponse(w)
+			if r.RequestURI == "/v0/user" {
+				next.ServeHTTP(w, r)
```

Comment:
It might be worthwhile coming up with an approach to abstract if an endpoint requires authorisation or not. But, for now, this will do.